src/components/text/Text.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
```

Comment:
Create a story book when that PR is merge as well

Reply:
Maybe it's better to merge this PR and add it to the storybook one?